docker-image.yml

```diff
@@ -0,0 +1,23 @@
+name: docker-hive CI
+
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag docker-hive:$(date +%s)
```

Comment:
it can use `docker/build-push-action@v2`:

```YAML
name: Build and push
        uses: docker/build-push-action@v2
        with:
          context: .
          push: false
          tags: docker-hive:latest
```

Reply:
Thanks for this over here! I actually made a mistake in opening this and somehow created a parallel branch in VSCode. Either way I will close here and reopen there.